HabitTracker.ConsoleApp-DreamFXX/Properties/launchSettings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "profiles": {
+    "HabitTracker_ConsoleApp": {
+      "commandName": "Project",
+      "workingDirectory": "C:\\Users\\marti\\source\\repos\\HabitTracker.ConsoleApp-DreamFXX",
```

Comment:
🟠 Hard-coded magic strings

💡 Remember that this path with exist on no one else's machine and therefore will need to be reconfigured.

🔧 A note in the README to highlight this prerequisite and to change this before launching the app.